README.md

```diff
@@ -52,22 +52,85 @@ This package exemplifies the use of the Mojaloop SDK Standard Components for TLS
 
 ## Inbound & Outbound API
 > This package delivers implementation Inbound and Outbound API services which will be used by Thirdparty to integrate with `Mojaloop Switch`
+  or used by a DFSP that wants to enabled Thirdparty functionality
+
+  - TODO: Separate and rename servers, handlers and models to be broken up Thirdparty ones and DFSP ones.
+  - TODO: Move models to `src/domain/stateMachine`
+  - TODO: Add a openapi file detailing what the DFSP backend needs to implement.
+         In some scenarios implementers will create a connector between the SDK and a DFSP's backend.
+### When used by a Thirdparty
+   Thirdparty's needs both the Outbound and Inbound service running.
+
+#### Inbound API
+   `Inbound API` service is called by `Mojaloop Switch`.
+   Its responsibility is to forward calls to `Thirdparty Backend` or help to deliver synchronous response for calls initiated by `Thirdparty backend` on `Outbound API`
+
+##### Deep Dive
+   Inbound handlers handle receiving of requests coming from the Switch.
+   These typically publish responses into redis for state machines to continue a flow.
+
+   Inbound models handle the processing of Switch requests but they themselves
+   can make requests to the accomplish a certain flow the initial inbound request initiates.
```

Comment:
Is there any reason to manually word wrap this? <-- comment applies through-out the readme changes.